### ISSUE TITLE
Why omnivore fails to fetch content in some webpages

### ISSUE THREAD

**Issue by natarslan** [omnivore-app/omnivore#3925]
I realise that once in a while omnivore cant fetch the article content both in Android app and in pc web browser. I was wondering if there are specific reasons behind this or is it just a random thing? 

The current fail is in this page:
https://climeworks.com/press-release/climeworks-switches-on-worlds-largest-direct-air-capture-plant-mammoth

**Comment by jacksonh** [comment]
Hey, that particular one seemed to fetch OK for me. Do you mind posting the error you see? 

Typically when we can't fetch a URL its some sort of blocking enabled by the site. Often it is CloudFlare "bot protection" which can be quite aggressive.

**Comment by natarslan** [comment]
Hey,

I double checked the web page. It actually works in PC (web browser now). But it still doesnt work on Android (version 14). Here comes the screenshot.

What I also checked:
- Made sure that NetGuard (vpn) is not blocking internet access to omnivore -> fetch fails
- Totally disabled NetGuard in case it was somehow interfering with Omnivore  -> fetch still fails

![omnivore](https://github.com/omnivore-app/omnivore/assets/15086016/93ea38ba-6a5d-4916-b88a-b76f0cced301)

**Comment by jacksonh** [comment]
When you say works on PC, you saved it with the browser extension there?

**Comment by natarslan** [comment]
Yes in desktop browser it works. But in android app the page didnt work (see the screenshot).

**Comment by jacksonh** [comment]
> Yes in desktop browser it works. But in android app the page didnt work (see the screenshot).

but you only saved the page once? Or you saved on Android also?

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
